src/patternfly/components/TextInputGroup/text-input-group.scss

```diff
@@ -16,6 +16,13 @@
   // Main
   --pf-c-text-input-group__main--first-child--not--text-input--MarginLeft: var(--pf-global--spacer--sm);
   --pf-c-text-input-group__main--m-icon__text-input--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-text-input-group__main--RowGap: var(--pf-global--spacer--xs);
+  --pf-c-text-input-group__main--ColumnGap: var(--pf-global--spacer--sm);
+
+  // Chip group
+  --pf-c-text-input-group__c-chip-group__main--PaddingTop: var(--pf-global--spacer--xs);
```

Comment:
With nested components, the common format is `--c-[component]` when its used as a child - https://www.patternfly.org/v4/developer-resources/global-css-variables#component-variables